modules/route53/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "region" {
```

Comment:
This will not correctly pick up the load balancer that is injected by Humanitec.

Inputs are supplied via one of `ip_address`, `ipv6_address`, `alias` or `name`.